### PR TITLE
Tutoritaledits

### PR DESCRIPTION
**Pull request by akaro2424**
Jen I did a bunch of edits for consistency for the tutorial. Take a look. 
I also now added with this branch a color picker section for my coloring section. I also added an eraser tool under the remove.md file (not sure if I did that in this branch or the master directly)

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
...
-5. Select __Bun Color__ to make it an active layer. 
+5. Select __Bun Color__ layer to make it an active layer. 
 6. Go to __File > Open...__ and open the saved burger image reference.
-7. Select __Color Picker__ to pick the bun color from the reference image and switch back to the burger project under the __Image Window__. You can create and modify your own color by double-clicking on the primary or secondary color boxes under [__Palette__](palette.md).
+7. [Select __Color Picker__ tool](coloring.md#select-an-exact-color-match-from-an-image) to pick the bun color from the reference image and switch back to the burger project under the __Image Window__. You can create and modify your own color by double-clicking on the primary or secondary color boxes under [__Palette__](palette.md).
 8. Select __Paint Brush__ to color in the entire burger. You can enlarge the size of the __Paint Brush__ in the __Tools Settings Bar__ to speed up the process.
-9. Select __Eraser__ to clean up any color that doesn't belong. 
-10. Save your progress and it should look like this: 
+9. Select the [__Eraser__ tool](remove.md#eraser) to clean up any color that doesn't belong.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original because the eraser was also introduced at the beginning already.

File: docs/tutorial.md [hunk 2]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
...
+9. Select the [__Eraser__ tool](remove.md#eraser) to clean up any color that doesn't belong. 
+10. Save your progress and it should look something like this: 
 ![color bun](img/burger/color_bun.gif)
 
 ### __Cheese, Tomatoes & Lettuce__ ###
 We'll now work on the cheese, tomatoes, and lettuce:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Bun Color__ layers.
-2. Open __Layer Properties__ for the new layer and enter __Other Color__ in the text box. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Other Color__.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original but keep in "dialog box"

File: docs/tutorial.md [hunk 3]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
...
 
 ### __Cheese, Tomatoes & Lettuce__ ###
 We'll now work on the cheese, tomatoes, and lettuce:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Bun Color__ layers.
-2. Open __Layer Properties__ for the new layer and enter __Other Color__ in the text box. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Other Color__. 
 3. Select __OK__ to close the window. 
-4. Select __Other Color__ to make it an active layer. 
+4. Select __Other Color__ layer to make it an active layer.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original

File: docs/tutorial.md [hunk 4]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
...
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Bun Color__ layers.
-2. Open __Layer Properties__ for the new layer and enter __Other Color__ in the text box. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Other Color__. 
 3. Select __OK__ to close the window. 
-4. Select __Other Color__ to make it an active layer. 
+4. Select __Other Color__ layer to make it an active layer. 
 5. Using the same concept as the bun, select __Color Picker__ to pick the colors from the reference image to apply to the project for the cheese, tomatoes, and lettuce. 
-6. Select __Paint Brush__ and __Eraser__ to color in the appropriate areas. 
+6. Select the __Paint Brush__ and __Eraser__ tools to color in the appropriate areas.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original

File: docs/tutorial.md [hunk 5]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
...
+6. Select the __Paint Brush__ and __Eraser__ tools to color in the appropriate areas. 
 7. Save your progress and it should look like this: 
 ![other color](img/burger/other_color.png)
 
 ### __Patty__ ###
 For the patty, we want to create a gradient to replicate the grilled side of the patty:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Other Color__ layers.
-2. Open __Layer Properties__ for the new layer, enter __Patty Color__, and enter opacity to 90. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Patty Color__ and enter an **Opacity** value of 90.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original but keep "dialog box" and the **opacity** part

File: docs/tutorial.md [hunk 6]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
...
 
 ### __Patty__ ###
 For the patty, we want to create a gradient to replicate the grilled side of the patty:
 
 1. Create a new layer. Make sure this new layer is *below* __Ink__ and *above* __Other Color__ layers.
-2. Open __Layer Properties__ for the new layer, enter __Patty Color__, and enter opacity to 90. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Patty Color__ and enter an **Opacity** value of 90. 
 3. Select __OK__ to close the window. 
-4. Select __Patty Color__ to make it an active layer. 
+4. Select __Patty Color__ layer to make it an active layer.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original

File: docs/tutorial.md [hunk 7]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
...
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Patty Color__ and enter an **Opacity** value of 90. 
 3. Select __OK__ to close the window. 
-4. Select __Patty Color__ to make it an active layer. 
+4. Select __Patty Color__ layer to make it an active layer. 
 5. Using the same concept as earlier, select __Color Picker__ to pick the patty color from the reference image and switch back to the project. 
 6. Right-click the black color in the __Palette__ to set the secondary color and select __Gradient__. 
 7. Click and hold on the drawing in any area while moving the cursor in different directions to form a gradient. Let go once you're satisfied with the way the color is distributed. Repeat if necessary.
 8. Select __Eraser__ to erase anything that is outside of the patty. Resize __Eraser__ to get the required size.
-9. Double-click on __Patty Color__ and enter opacity to 100. Select __OK__ to close the window. 
+9. Double-click on __Patty Color__ layer in the **Layers** window and enter **Opacity** value of 100. Select __OK__ to close the window.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original because this is too wordy

File: docs/tutorial.md [hunk 8]
```diff
@@ -156,19 +157,19 @@ From here, we can start to add these elements against the sketch layer again.
   
 1. Create a new layer. Make sure this new layer is *below* __Ink__ layer.
 2. Check the box for __Sketch__ in the __Layers Window__ to [unhide](edit.md#hide/unhide-layers) the sketch.
-3. Select __Ellipse__ and __Line__ to create a speech bubble over the sketched speech bubble.
-5. Select __Eraser__ to erase a part of the ellipse that's connected to the lines to complete the speech bubble.
-6. Select __Text__ and modify the font in the __Tools Settings Bar__.
+3. [Select __Ellipse__](objects.md) and [select __Line__](objects.md) to create a speech bubble over the sketched speech bubble.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original but add in the links

File: docs/tutorial.md [hunk 9]
```diff
@@ -156,19 +157,19 @@ From here, we can start to add these elements against the sketch layer again.
   
 1. Create a new layer. Make sure this new layer is *below* __Ink__ layer.
 2. Check the box for __Sketch__ in the __Layers Window__ to [unhide](edit.md#hide/unhide-layers) the sketch.
-3. Select __Ellipse__ and __Line__ to create a speech bubble over the sketched speech bubble.
-5. Select __Eraser__ to erase a part of the ellipse that's connected to the lines to complete the speech bubble.
-6. Select __Text__ and modify the font in the __Tools Settings Bar__.
+3. [Select __Ellipse__](objects.md) and [select __Line__](objects.md) to create a speech bubble over the sketched speech bubble.
+5. [Select __Eraser__](remove.md#eraser) to erase a part of the ellipse that's connected to the lines to complete the speech bubble.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original but add in the links

File: docs/tutorial.md [hunk 10]
```diff
@@ -156,19 +157,19 @@ From here, we can start to add these elements against the sketch layer again.
   
 1. Create a new layer. Make sure this new layer is *below* __Ink__ layer.
 2. Check the box for __Sketch__ in the __Layers Window__ to [unhide](edit.md#hide/unhide-layers) the sketch.
-3. Select __Ellipse__ and __Line__ to create a speech bubble over the sketched speech bubble.
-5. Select __Eraser__ to erase a part of the ellipse that's connected to the lines to complete the speech bubble.
-6. Select __Text__ and modify the font in the __Tools Settings Bar__.
+3. [Select __Ellipse__](objects.md) and [select __Line__](objects.md) to create a speech bubble over the sketched speech bubble.
+5. [Select __Eraser__](remove.md#eraser) to erase a part of the ellipse that's connected to the lines to complete the speech bubble.
+6. [Select __Text__](text.md) and modify the font in the __Tool__ bar settings.
```
Comment (jeneira94) [inline docs/tutorial.md]: keep original but add in the links and the tools settings bar is what is already named in the overview so we have to see what naming convention we prefer

**Review by jeneira94 — CHANGES_REQUESTED** [review]
i can make the changes after since we have discussed it already. i had to say keep original for a lot of things because this part is differnt from the rest of the other .md pages. this tutorial is a workflow so theres a lot of wordiness that can be eliminated onces the tools and commands have been presented once since this is a guide that people have to do all in one go, not like the others where they can navigate and go to whichever areas they need at the time.

File: docs/coloring.md [hunk 11]
```diff
@@ -3,7 +3,9 @@ To add color to an object/text/image/layer you will need to select a primary and
 
 You can quickly select a color from the [color palette](palette.md) (left click on chosen color in palette to select primary and right click to select secondary) or in some cases you may want to choose a more customized color.
```
Comment (jeneira94) [inline docs/coloring.md]: > (left click on chosen color in palette to select primary and right click to select secondary

i would write:

> (left-click on the palette color select primary, and right-click to select secondary)

File: docs/coloring.md [hunk 12]
```diff
@@ -3,7 +3,9 @@ To add color to an object/text/image/layer you will need to select a primary and
 
 You can quickly select a color from the [color palette](palette.md) (left click on chosen color in palette to select primary and right click to select secondary) or in some cases you may want to choose a more customized color.
 
-To select a customize color:
+To select a customize color you can set the customized color by either using **Set Color Values** or **Select an Exact Color Match From an Image** methods.
```
Comment (jeneira94) [inline docs/coloring.md]: > To select a customize color you can set the customized color by either using **Set Color Values** or **Select an Exact Color Match From an Image** methods.

I would write:

> Select a customized color by using **Set Color Values** or **Select an Exact Color Match From an Image**.

File: docs/tutorial.md (hunk 6 above)
Comment (akaro2424) [reply docs/tutorial.md]: This could be confusing for someone as they could click on the Patty!! not the Patty Color Layer... wonder if you are too close to creating the instructions to notice this...but it was confusing when I read it.

File: docs/coloring.md [hunk 13]
```diff
@@ -3,7 +3,9 @@ To add color to an object/text/image/layer you will need to select a primary and
 
 You can quickly select a color from the [color palette](palette.md) (left click on chosen color in palette to select primary and right click to select secondary) or in some cases you may want to choose a more customized color.
 
-To select a customize color:
+To select a customize color you can set the customized color by either using **Set Color Values** or **Select an Exact Color Match From an Image** methods.
+
+## Set Color Values
 
 1. Click on the *Primary Color* or *Secondary Color* boxes in the Palette. You can click the switch double arrow symbol to switch between the primary and secondary colors.
```
Comment (jeneira94) [inline docs/coloring.md]: > 1. Click on the *Primary Color* or *Secondary Color* boxes in the Palette. You can click the switch double arrow symbol to switch between the primary and secondary colors.

i would write:

> 1. Select the **Primary Color** or **Secondary Color** boxes in the **Palette**. Click the double-arrow toggle to switch between the primary and secondary colors.

File: docs/tutorial.md (hunk 7 above)
Comment (akaro2424) [reply docs/tutorial.md]: opacity to 100 does not make sense grammatically

File: docs/tutorial.md (hunk 10 above)
Comment (akaro2424) [reply docs/tutorial.md]: Your general overview diagram upfrontin the Pinta Workspace says Toolbar

File: docs/coloring.md [hunk 14]
```diff
@@ -37,7 +39,17 @@ To select a customize color:
     -  **Red**, **Green**, **Blue**: Enter the RGB Color. The triangle dial selector will move then to the selected RGB value.
     -  **Opacity**: Enter in the value desired (0 - 255). The lower the number chosen the more translucent the color will be.
     -  **Color Name**: If you know the color name you want you can directly input the value. Entering in a number will change the RGB color and move the dial selector settings to the corresponding color. If the RGB color or the triangle dial selector was used to select a color this color name value will be updated.  
-    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color.  An equivalent "Eye Dropper" type tool can also be accessed from the **Tools** menu (referenced as a Color Picker ![color picker](img/overview/picker.png)). 
+    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color. The corresponding color values will then be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog box.
```
Comment (jeneira94) [inline docs/coloring.md:42]: >     -  **Eye Dropper**: Click on the eyedropper eye dropper and then click a color anywhere on your screen to select that color. The corresponding color values will then be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog box.  

I would write:

>     -  **Color Picker**: Select the **Color Picker** eye dropper and then choose a color anywhere on your screen to duplicate that color. The corresponding color values will be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog boxes.

File: docs/coloring.md [hunk 15]
```diff
@@ -37,7 +39,17 @@ To select a customize color:
...
     -  **Color Name**: If you know the color name you want you can directly input the value. Entering in a number will change the RGB color and move the dial selector settings to the corresponding color. If the RGB color or the triangle dial selector was used to select a color this color name value will be updated.  
-    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color.  An equivalent "Eye Dropper" type tool can also be accessed from the **Tools** menu (referenced as a Color Picker ![color picker](img/overview/picker.png)). 
+    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color. The corresponding color values will then be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog box.  
+
+## Select an Exact Color Match From an Image  
+    
+   1. Select the *Color Picker* tool ![color picker](img/overview/picker.png) from the **Tools** menu
```
Comment (jeneira94) [inline docs/coloring.md]: >    1. Select the *Color Picker* tool color picker from the **Tools** menu

I would write:

>    1. Select the **Color Picker** color picker from the **Tools** menu.

File: docs/coloring.md [hunk 16]
```diff
@@ -37,7 +39,17 @@ To select a customize color:
...
-    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color.  An equivalent "Eye Dropper" type tool can also be accessed from the **Tools** menu (referenced as a Color Picker ![color picker](img/overview/picker.png)). 
+    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color. The corresponding color values will then be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog box.  
+
+## Select an Exact Color Match From an Image  
+    
+   1. Select the *Color Picker* tool ![color picker](img/overview/picker.png) from the **Tools** menu
+   
+      ![eye dropper](img/color/colorpicker.png).
+   
+   2. Left click to set the primary color and right click to set the secondary color.
```
Comment (jeneira94) [inline docs/coloring.md]: >    2. Left click to set the primary color and right click to set the secondary color.

I would write:
>    2. Click to set the primary color and right-click to set the secondary color.

File: docs/coloring.md [hunk 17]
```diff
@@ -37,7 +39,17 @@ To select a customize color:
...
+   3. Drag the *Color Picker* tool ![color picker](img/overview/picker.png) across the image and click on the image color pixel you would like to capture.
```
Comment (jeneira94) [inline docs/coloring.md]: 3. Move the **Color Picker** color picker across the workspace and click on the image color pixel you would like to capture.

**Review by jeneira94 — CHANGES_REQUESTED** [review]
There are some wordy sentences and I changed the term to color picker instead of eyedropper.

File: docs/tutorial.md [hunk 18]
```diff
@@ -95,42 +96,42 @@ This is now the fun part: coloring! For the purposes of this tutorial, we also a
 ### __Bun__ ###
 Now let's begin with the bun:
 
-1. Uncheck the box for __Sketch__ in the __Layers Window__ to [hide](edit.md#hide/unhide-layers) the sketch from view. We won't be needing it for this process.
+1. Uncheck the box for __Sketch__ in the __Layers Window__ to [hide the layer](edit.md#hide/unhide-layers) "Sketch" from view. We won't be needing it for this process.
```
[the remark this answers is not in the document]
Comment (akaro2424) [reply docs/tutorial.md]: I think links should more descriptive than just a word like hide (I think Jamie has an issue with this and I do not think it is good practice with links)

File: docs/tutorial.md (hunk 6 above)
Comment (jeneira94) [reply docs/tutorial.md]: but that's because i put the note at the top to read the setup page first to understand. this is also a perspective of when some doesnt understand the concept, we have mentioned to check out the layer concept. from there, they should be able to see what an active layer mean and implies. let me know your thoughts. i;m also not too against putting the word there if you feel its necessary. i just thought its too wordy because i thought its obvious from doing the setup page already.

File: docs/tutorial.md (hunk 7 above)
Comment (jeneira94) [reply docs/tutorial.md]: oh nvm i agree. i don't know why i wrote i was keeping the original. maybe i got mixed up with another line. here. this part is good.

File: docs/tutorial.md (hunk 10 above)
Comment (jeneira94) [reply docs/tutorial.md]: no the toolbar is different. please see the overview page again. the tools settings bar is something else. if you go to overview.md > Tools Menu, you'll see the Tools Settings Bar. It's not the same as the Toolbar. This is the bar that you use to adjust your tools' properties as explained in the overview

File: docs/tutorial.md [hunk 19]
```diff
@@ -69,13 +70,13 @@ The tools we'll use for inking are:
 This part of the process, we'll be inking the rough sketch itself so that all the lines needed are more distinct.
 
 1. Create a new layer in the __Layers Window__. Make sure this new layer is *above* __Color Sketch__ layer. 
-2. Open __Layer Properties__ for the new layer and enter __Ink__ in the text box. 
+2. Double-click the new layer to open __Layer Properties__ dialog box and under the name type __Ink__. 
 3. Select __OK__ to close the window.  
-4. Select __Ink__ to make it an active layer. 
-5. Select __Paint Brush__ to start inking over your rough sketches. 
+4. Select __Ink__ layer to make it an active layer.
```
[the remark this answers is not in the document]
Comment (akaro2424) [reply docs/tutorial.md]: I still think this sounds strange to not put in the word layer... the user may think they are selecting a type of ink??

File: docs/coloring.md (hunk 14 above)
Comment (akaro2424) [reply docs/coloring.md:42]: No - this is incorrect - there is a difference between the eye dropper and color picker. The eye dropper is in the Choose Primary Color window that pops up when selecting colors and the Color picker is the tool on the tool menu. That is why I partly made a new heading for the Color picker ... you need to see the whole color section to understand this-

File: docs/tutorial.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (jeneira94) [reply docs/tutorial.md]: i agree

File: docs/coloring.md (hunk 14 above)
Comment (jeneira94) [reply docs/coloring.md:42]: ahhhhhh i see what you mean now
